spark/src/main/resources/META-INF/services/org.apache.spark.sql.sources.DataSourceRegister

```diff
@@ -1 +1,2 @@
-org.apache.spark.sql.delta.sources.DeltaDataSource
\ No newline at end of file
+org.apache.spark.sql.delta.sources.DeltaDataSource
+io.delta.sharing.spark.DeltaSharingDataSource
```

Comment:
shouldn't this be part of the `sharing` module? What happens when just the sharing module (and the `DeltaSharingDataSource`) is not part of the class path?

Reply:
Are you suggesting another DataSourceRegister in the sharing module?

Reply:
@vkorukanti 
the unit tests did complain: 
```
[info]   java.util.ServiceConfigurationError: org.apache.spark.sql.sources.DataSourceRegister: Provider io.delta.sharing.spark.DeltaSharingDataSource not found
```